src/Windows/curl_easy.h

```diff
@@ -40,7 +40,8 @@ class curl_easy
     static std::unique_ptr<curl_easy> create(
         const std::string& url,
         const std::string* const p_body,
-        unsigned long dwFlags = WINHTTP_FLAG_SECURE);
+        unsigned long dwFlags = WINHTTP_FLAG_SECURE,
+        bool fetchFromLocalAgent = false);

```

Comment:
I think that fetchFromLocalAgent should be called fetchingFromLocalAgent instead, to better signal that we're notifying curl that he's fetching from the local agent, rather than telling curl to go fetch from the local agent.

Reply:
done